Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml

```diff
@@ -676,6 +676,25 @@
   - type: Sprite
     state: slime
 
+- type: entity
+  name: raw snail meat
+  parent: FoodMeatRawBase
+  id: FoodMeatSnail
```

Comment:
This should have FoodSequenceElement and the corresponding translation strings so it can be added to custom burgers and tacos.
Merge master to see how the other meat types do it.